### ISSUE TITLE
INFO - Message sent to number but it no longer arrives to WA

### ISSUE THREAD

**Issue by george72** [Neurotech-HQ/heyoo#83]
Just started testing heyoo/wa yesterday to send me text messages and it worked but today log shows message was sent succesfully but the message never arrived WA. 
I had to renew token in Could API (token error was in the log but still message say it was sent succesfully but it wasn't. Even after renewing and using new temporary token it does not work. Sending a message directly from Meta works.  Sending a message from heyoo says it was sent but it did not arrive to my WA.

2023-05-24 21:26:53,328 - root - INFO - Sending message to 520000000000
2023-05-24 21:26:54,132 - root - INFO - Message sent to 520000000000

Using Fedora 38, Python 3.10.11

**Comment by filipporomani** [comment]
To explain a bit more, a message is not received by the user in the following 2 conditions:

- the message isn't an approved template message
OR
- the user didn't send a message to your business phone number in the past 24 hours.

You can fix this issue in 2 ways:
- sending a template instead of a custom message (remember that you can use variables in templates)
- sending a message to you business phone number and then sending as many messages as you want. 

I personally suggest you to use templates, as no user interaction is needed.

**Comment by george72** [comment]
If user send a message to my business phone, all messages arrives to the user. I will try templates too as it does not requires user interaction.

However when message was not received I did not get any error message, in fact I got a success message. It would be great to get an error instead of a success when messages were not arriving.

**Comment by filipporomani** [comment]
I don't think that an error is given with the current API version (14). I alr opened a PR to switch to v17 (latest), but I'm still not sure it gives an error tho.
